test/cli_run_network_test.go

```diff
@@ -33,3 +35,18 @@ func (suite *PouchRunNetworkSuite) TestRunWithPing(c *check.C) {
 	defer DelContainerForceMultyTime(c, name)
 	res.Assert(c, icmd.Success)
 }
+
+// TestRunWithPublicAll is to verify run container with publish-all flag
+func (suite *PouchRunNetworkSuite) TestRunWithPublishAll(c *check.C) {
```

Comment:
Could you also add a test case for testing an image which expose multiple ports? @ZYecho

Reply:
ok, and could you help to check to CI result? it looks strange and I don't know why it failed.

Reply:
done.